webapp/publisher/snaps/logic.py

```diff
@@ -25,6 +25,13 @@ def get_snaps_account_info(account_info):
 
     now = datetime.datetime.utcnow()
 
+    for snap in user_snaps:
+        snap_info = user_snaps[snap]
+        for revision in snap_info["latest_revisions"]:
+            if len(revision["channels"]) > 0:
+                snap_info["latest_revision"] = revision
```

Comment:
The name may be a bit confusing.
Technically "latest_revision" is `latest_revisions[0]`, the one we are interested in is latest released revision. So I'd be for calling it exactly what it is, or at least `latest_release` for short.

Reply:
Much better name :+1: